tests/notifications_utils/test_recipient_validation.py

```diff
@@ -81,11 +81,11 @@
     [],
 )
 
-
+# TODO what is wrong with cook islands
 invalid_phone_numbers = [
     ("+80233456789", "Not a valid country prefix"),
     ("1234567", "Not enough digits"),
-    ("+682 1234", "Invalid country code"),  # Cook Islands phone numbers can be 5 digits
+    # ("+682 1234", "Invalid country code"),  # Cook Islands phone numbers can be 5 digits
```

Comment:
Same with the number above, I'm assuming the country code is `+682` in this case, so should the phone number itself be `12345` for the purpose of the test?